entrypoint.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+# check codecov token exist.
+if [[ ! -n $PLUGIN_TOKEN ]] && [[ ! -n $CODECOV_TOKEN ]]; then
+    echo "missing codecov token"
```

Comment:
Please change the message to be consistent with the others `echo "-- Error: missing Codecov token"`